Gemfile

```diff
@@ -69,6 +69,9 @@ gem 'geokit-rails'
 # activity seed
 gem 'public_activity'
 
+# messaging
+gem 'mailboxer', github: 'lacco/mailboxer'
```

Comment:
Assuming this is using a custom mailboxer repo to support Rails 4.2 - let me know if otherwise.